### PR TITLE
Restructure Codebase #35

### PR DESCRIPTION
**Pull request by sparkle-sparkle**
replace document.getElementById() with jquery selector

### CODE REVIEW

**Comment by StevenCopeland** [comment]
Hello, @sparkle-sparkle! This is your first Pull Request that will be reviewed by [Ebert](https://ebertapp.io/), an automatic Code Review service. It will leave comments on this diff with potential issues and style violations found in the code as you push new commits. You can also see all the issues found on this Pull Request on its [review page](https://ebertapp.io/github/StevenCopeland/FILEPLAN/pulls/37). Please check our [documentation](https://docs.ebertapp.io) for more information.

File: script.js [hunk 1]
```diff
@@ -191,17 +191,17 @@ var invalid = function() {
 
 var spaceGenPerSec = function() {
 	setInterval(function() {
-		var second = document.getElementById("perSecond").innerHTML = "per second: " + clicker.perSec + "";
+		var second = $("#perSecond").innerHTML = "per second: " + clicker.perSec + "";
 		clicker.bytes = sumDecimals(clicker.bytes, clicker.perSec);
-		var perSec = document.getElementById("bytes").innerHTML = "Bytes: " + clicker.bytes + "";
+		var perSec = $("#bytes").innerHTML = "Bytes: " + clicker.bytes + "";
```
Comment (StevenCopeland) [inline script.js]: Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)

Comment (StevenCopeland) [inline script.js]: Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)

Comment (StevenCopeland) [inline script.js]: Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)

Comment (StevenCopeland) [inline script.js]: Possible undefined variable: '$' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 2]
```diff
@@ -191,17 +191,17 @@ var invalid = function() {
 
 var spaceGenPerSec = function() {
 	setInterval(function() {
-		var second = document.getElementById("perSecond").innerHTML = "per second: " + clicker.perSec + "";
+		var second = $("#perSecond").innerHTML = "per second: " + clicker.perSec + "";
```
Comment (StevenCopeland) [inline script.js]: Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)

Comment (StevenCopeland) [inline script.js]: Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)

Comment (StevenCopeland) [inline script.js]: Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)

Comment (StevenCopeland) [inline script.js]: Possible undefined variable: '$' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 3]
```diff
@@ -145,7 +145,7 @@ var showUpgrades = function(amount, name) {
 // showCode main function
 var showCode = function(amount, id, name) {
 	if (clicker.bytes >= amount) {
-		var idea = document.getElementById(id).innerHTML = name;
+		var idea = $('#' + id).innerHTML = name;
```
Comment (StevenCopeland) [inline script.js]: Possible undefined variable: '$' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 4]
```diff
@@ -131,7 +131,7 @@ function appendText(space, bytes) {
 
 var dateFunc = function() {
 	setInterval(function() {
-		document.getElementById("dateCount").innerHTML = Date();
+		$("#dateCount").innerHTML = Date();
```
Comment (StevenCopeland) [inline script.js]: Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)

Comment (StevenCopeland) [inline script.js]: Possible undefined variable: '$' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 5]
```diff
@@ -36,19 +36,19 @@ fillByteUpgradeSec()
 		{
 			byteUpgradeIdeaArray.push("#byteUpgradeIdea"+counter);
 			counter++
-		}	
+		}
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 0 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 6]
```diff
@@ -248,7 +248,7 @@ secondFunction();
 
 var spaceGen = function() {
 	clicker.bytes = sumDecimals(clicker.bytes, clicker.perClick);
-	var count = document.getElementById("bytes").innerHTML = "Bytes: " + clicker.bytes + "";
+	var count = $("#bytes").innerHTML = "Bytes: " + clicker.bytes + "";
```
Comment (StevenCopeland) [inline script.js]: Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)

Comment (StevenCopeland) [inline script.js]: Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)

Comment (StevenCopeland) [inline script.js]: Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)

Comment (StevenCopeland) [inline script.js]: Possible undefined variable: '$' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

**Comment by StevenCopeland** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 fixed issue! 🎉

You can see more details about this review at https://ebertapp.io/github/StevenCopeland/FILEPLAN/pulls/37.

**Comment by StevenCopeland** [comment]
@sparkle-sparkle I reverted this because I tested the code and it did not work. Try running it on your repo(enable github pages) and give me the link so I can make sure it works.

**Comment by sparkle-sparkle** [comment]
Here is the URL https://sparkle-sparkle.github.io/FILEPLAN/

**Comment by StevenCopeland** [comment]
Do you see the bug?

**Comment by sparkle-sparkle** [comment]
I saw that it's not working. I'll have to look at it later.

**Pull request merged**